pkg/agent/route/route_linux.go

```diff
@@ -24,6 +24,7 @@ import (
 	"time"
 
 	"github.com/containernetworking/plugins/pkg/ip"
+	goiptables "github.com/coreos/go-iptables/iptables"
```

Comment:
I think it was a good idea to "hide" this implementation-specific package and only import it in `antrea.io/antrea/pkg/agent/util/iptables`.

Reply:
Used the Protocol struct in antrea's iptables.go.